lib/tasks/export_local_transactions.rake

```diff
@@ -0,0 +1,15 @@
+desc "Export Local Transactions' slugs, names, LGSL and optional LGIL codes as CSV"
+
+task :export_local_transactions => :environment do
+  require "csv"
+
+  csv_string = CSV.generate do |csv|
+    csv << ["slug","lgsl","lgil","title"]
+
+    Edition.where(_type: "LocalTransactionEdition").each do |lte|
+      csv << [lte.slug,lte.lgsl_code,lte.lgil_override,lte.title]
```

Comment:
It's worth noting that `lgil_override` is unset for most local transactions.  If not set, there's [some logic](https://github.com/alphagov/govuk_content_models/blob/master/app/models/local_authority.rb#L41-L53) to pick a LGIL from the available ones. It picks the lowest lgil that's not 8 (information), but falls back to 8 if there are no others.